server/protocols/jmap/src/test/java/org/apache/james/jmap/utils/FilterToSearchQueryTest.java

```diff
@@ -170,7 +170,8 @@ public void filterConditionShouldMapWhenText() {
                 SearchQuery.address(AddressType.Cc, text),
                 SearchQuery.address(AddressType.Bcc, text),
                 SearchQuery.headerContains("Subject", text),
-                SearchQuery.bodyContains(text))));
+                SearchQuery.bodyContains(text),
+                SearchQuery.attachmentContains(text))));
```

Comment:
that's strange to see a commit without test :) I guess integration test are following...

Reply:
That's it